### PR TITLE
Created Initial write up of globals.md

### LINKED ISSUE #2143: [Term Entry] Python Built-in Functions globals()
### Reason/inspiration (optional)

We would like a new term entry on the `globals()` Python: [Built-in Function](https://www.codecademy.com/resources/docs/python/built-in-functions). The entry should go in a new file under docs/content/python/concepts/built-in-functions/terms/globals/globals.md

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ```pseudo code snippet.
- An "Example" section with a ```py code snippet.
- A “Codebyte Example” section with a ```codebyte/python snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by mossj22**
### Description

Created new Entry for globals(). Provided two examples of globals().
Closes #2143 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (dakshdeepHERE) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:19]:
```suggestion
The built-in **globals()** function allows access to the global scope's name table, which is a writable dictionary containing the current global names and their corresponding values in the code. This function can be used to access or modify the value of a global variable from within functions.
```
Try not to use pronouns.

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (dakshdeepHERE) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:46]:
```suggestion
```

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (dakshdeepHERE) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:52]:
```suggestion
```

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (dakshdeepHERE) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:60]:
```suggestion
The code will return:
```

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (dakshdeepHERE) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:68]:
```suggestion
Use `globals()` to get the symbol table:
```

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (dakshdeepHERE) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:72]:
```suggestion
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side.

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:30]:
```suggestion

```
Remove HTML and replace with a blank line.

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:48]:
```suggestion

### Example 2
```
Blank line before and after headers.

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:58]:
```suggestion
The global variable `length` is set to `123`. The global variable in this example can be modified using the `globals()` function with the dictionary key `[length]`. It can be also modified from within a function.
```

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [reply content/python/concepts/built-in-functions/terms/globals/globals.md:19]: In addition to avoiding first- and second- person pronouns, put single backticks around `globals()` as well as bolding it:

```
**`globals()`**
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@mossj22 I did a maintainer review and had a few comments to add to @dakshdeepHERE's comments.

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:29]:
```suggestion
The `globals()` method returns the dictionary of the current global symbol table.
```

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:42]:
```suggestion
This code returns the following dictionary. Notice that the `length` global variable is listed in the dictionary.
```

**Comment by mossj22** [comment]
Hello, i made all the changes as requested. Many thanks

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:45]:
```suggestion
{'__name__': '__main__', '__doc__': None, '__package__': None, '__loader__': <class '_frozen_importlib.BuiltinImporter'>, '__spec__': None, '__annotations__': {}, '__builtins__': <module 'builtins' (built-in)>, 'length': 123}
```

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:42]:
```suggestion
This code returns the following dictionary. Notice that the `length` global variable is listed in the dictionary.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@mossj22 I had a few more comments.

File: content/python/concepts/built-in-functions/terms/globals/globals.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/globals/globals.md:39]:
```suggestion
```
Remove extra blank line.

**Review by SSwiniarski — APPROVED** [review]
@mossj22 Looks good for second review.

**Comment by SSwiniarski** [comment]
@mossj22 Note you also still have to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=2537).

**Comment by mossj22** [comment]
@SSwiniarski I have signed the CLA.

**Review by caupolicandiaz — APPROVED** [review]
@mossj22 @SSwiniarski I just added some minor formatting edits. Looks good to merge.

**Comment by SSwiniarski** [comment]
@mossj22 Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/python/built-in-functions/globals

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**
